src/App.css

```diff
@@ -691,11 +692,22 @@ nav {
 .header {
   align-items: center;
   display: grid;
-  grid-template-columns: 2fr 1fr;
+  grid-template-columns: 2fr minmax(min-content, 2rem);
```

Comment:
Nice

Reply:
Yep! Yesterday, I learned a little about CSS Grid and this amazing function `minmax` 😄